kn/fotos/entities.py

```diff
@@ -71,7 +72,7 @@ def id(self):
     def required_visibility(self, user):
         if user is None:
             return frozenset(('world',))
-        if 'webcie' in user.cached_groups_names:
+        if user == 'system' or 'webcie' in user.cached_groups_names:
```

Comment:
`user` is normaal gesproken een object en geen string.
Maar ik zie dat je dit al gemergd hebt.